CorsixTH/Lua/world.lua

```diff
@@ -1378,10 +1378,12 @@ end
 -- PRIVATE method to generate the next VIP date
 function World:_generateNextVipDate()
   -- Support standard values for mean and variance
-  local mean = 180
+  local mean = self.map.level_config.gbv.MayorLaunch
   local variance = 30
   -- How many days until next vip?
   local days = math.round(math.n_random(mean, variance))
+  -- Add a safety net so the next_vip_date can't end up <= 0
```

Comment:
This comment might be too literal to the code line.
Perhaps "The VIP is not a time traveller" for fun or remove completely?

Reply:
The real culprit is likely that a normal distribution is the wrong way to compute this, a Poisson distribution seems much more appropriate: https://en.wikipedia.org/wiki/Poisson_distribution

That trivially solved the problem, as the range of the latter is `[0,infinity)` rather than `(-infinity, infinity)` of the normal distribution.

Reply:
Another option is to use the binomial distribution: https://en.wikipedia.org/wiki/Binomial_distribution which has similar properties as Poisson.